### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs [hunk 1]
```diff
@@ -317,9 +317,6 @@ instance Era era => ToObject (UtxowPredicateFailure era) where
              , "txBodyMetaDataHash" .= txBodyMetaDataHash
              , "fullMetaDataHash" .= fullMetaDataHash
              ]
-  toObject _verb InvalidMetaData =
-    mkObject [ "kind" .= String "InvalidMetaData"
-             ]
```
Comment (intricate) [inline cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs:322]: Why are we removing this case? It still appears to be a constructor of the `UtxowPredicateFailure era` type.

Comment (newhoggy) [reply cardano-node/src/Cardano/Tracing/OrphanInstances/Shelley.hs:322]: Not sure, the constructor is gone altogether.

**Comment by mrBliss** [comment]
Please see https://github.com/input-output-hk/cardano-node/pull/1928

**Comment by mrBliss** [comment]
@newhoggy Is this still relevant?

**Pull request closed without merging**